src/components/Navigation/Navigation.scss

```diff
@@ -25,13 +26,9 @@ $nav-max-width: rem(360px);
 
   @include breakpoint-after(nav-min-window-corrected()) {
     max-width: layout-width(nav);
-    border-right: var(--p-override-none, border());
+    // border-right: var(--p-override-none);
```

Comment:
Shouldn't this be `border-right: border();`

Reply:
Oh woops, this should be `divider` which is now declared above. We should remove this line?